draft-ietf-jsonpath-base.md

```diff
@@ -226,6 +226,9 @@ Normalized Path:
   providing a query that results in exactly that node.  Similar
   to, but syntactically different from, a JSON Pointer {{-pointer}}.
 
+Singular Path:
```

Comment:
The question is whether we define this via the outcome or by saying how it is constructed.  The ABNF of course does the latter.

Reply:
Normalized Path is defined via the outcome, so I think we should do the same here for consistency.

Reply:
I think I've understood @cabo's point better now. See 46ce76c95cd631faba3b33602f9d0e7175a73d13 for a possible solution.